tests/vulnerability/test_vulnerability.py

```diff
@@ -509,114 +510,125 @@ def check_raw_output(result):
             raw_res = result["raw_data"]
-            if downscaling_method == "Dynamical":
-                if approach == "Time":
-                    if historical_data == "Historical Reconstruction":
-                        assert raw_res.shape == (
-                            (end_year - start_year + 1) * days_by_season[season] * 24,
-                            batch,
-                        )  # Finding the total number of hours after filtering for specific seasons
-                    elif wrf_bias_adjust == True:
-                        num_sims = 5 * batch
-                        assert raw_res.shape == (
-                            1,
-                            num_sims,
-                            (end_year - start_year + 1) * days_by_season[season] * 24,
-                        )  # Finding the total number of hours after filtering for specific seasons
-                    elif wrf_bias_adjust == False:
-                        num_sims = 8 * batch
-                        assert raw_res.shape == (
-                            1,
-                            num_sims,
-                            (end_year - start_year + 1) * days_by_season[season] * 24,
-                        )  # Finding the total number of hours after filtering for specific seasons
-                elif approach == "Warming Level":
-                    assert raw_res.shape == (
-                        1,
-                        8 * batch,
-                        30
-                        * days_by_season[season]
-                        * 24,  # Pre-determined as 30-year WL slices; finding the total number of hours
-                    )
-                assert raw_res.name == variable
-            elif downscaling_method == "Statistical":
-                if approach == "Time":
-                    assert raw_res.shape == (
-                        1,
-                        62 * batch,
-                        (end_year - start_year + 1) * days_by_season[season],
-                    )  # Finding the total number of hours after filtering for specific seasons
-                elif approach == "Warming Level":
-                    assert raw_res.shape == (
-                        1,
-                        30
-                        * days_by_season[
-                            season
-                        ],  # Pre-determined as 30-year WL slices; finding the total number of days
-                        129 * batch,
-                    )
-                # Make sure that raw dataset pulls the correct, re-defined LOCA variable for `Air Temperature at 2m`
-                if metric_calc == "max" and variable == "Air Temperature at 2m":
-                    assert raw_res.name == "Maximum air temperature at 2m"
-                elif metric_calc == "min" and variable == "Air Temperature at 2m":
-                    assert raw_res.name == "Minimum air temperature at 2m"
+            match downscaling_method:
```

Comment:
match on multiple:
`match (downscaling_method, approach):`

Reply:
Changed in 88b2baa61ea1912777602d59c66c9786dae41d0e